tests/test_database_manager.py

```diff
@@ -0,0 +1,73 @@
+"""
+test_database_manager.py - UT for the databaseManager
+
+Copyright (c) 2018 The Fuel Rats Mischief,
+All rights reserved.
+
+Licensed under the BSD 3-Clause License.
+
+See LICENSE.md
+
+This module is built on top of the Pydle system.
+"""
+from unittest import TestCase, skip
+from Modules.database_manager import DataBaseManager
+
+
+class TestDB(TestCase):

```

Comment:
No problem, thats why I am informing you now. Thankfully you only have a handful of tests so it will be relatively easy to convert them. The key difference between `unittest` and `pytest` is that PyTests do not need to be in a class (as unittest requires tests be in a subclass of TestCase), and that PyTests make use of bare assertions. 

While `unittest` designed tests will work under pytest, it is undesirable to add additional tests targeting that framework as you cannot make good use of PyTests advanced functionalities.

Reply:
Hopefully done